python/fledge/services/core/api/plugins/remove.py

```diff
@@ -208,6 +208,8 @@ def purge_plugin(plugin_type: str, name: str) -> tuple:
         code = os.system(cmd)
         if code:
             raise PackageError(link)
+        else:
+            common._get_available_packages.cache_clear()
```

Comment:
Also update the _package_cache_manager list attributes (like last_accessed_time)